### PR TITLE
Added optional 'db' command line parameter to fetch.go example with 'protein' as the default.

### PR DESCRIPTION
**Pull request by daniel-anavaino**
Minor change to the fetch.go example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,12 +44,12 @@ import (
 )
 
 const (
-	db   = "protein"
 	tool = "entrez.example"
 )
 
 var (
 	clQuery = flag.String("query", "", "query specifies the search query for record retrieval (required).")
+	db      = flag.String("db", "protein", "db specifies the database to search")
```
Comment (kortschak) [inline README.md:52]: Probably you should check `*db` is not `""` at line 70 and add a "(required)" here.

Comment (daniel-anavaino) [reply README.md:52]: Doesn't the default take care of that?

Comment (daniel-anavaino) [reply README.md:52]: If you don't use the option, you get protein by default.

Comment (kortschak) [reply README.md:52]: Sorry, you're correct.

**Review by kortschak — APPROVED** [review]

**Pull request merged**